src/main/java/core/basesyntax/DataConverterImpl.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
```

Comment:
The class `DataConverterImpl` is directly implementing `DataConverter` which might not be ideal if you plan to add more conversion methods in the future. Consider if it's beneficial to have an interface for each conversion type to follow the Interface Segregation Principle from SOLID.